newdle/client/src/client.js

```diff
@@ -65,18 +65,57 @@ class Client {
           try {
             const result = await func.bind(this)(...params);
             return await prevPromise(result);
-          } catch (exc) {
-            dispatch({type: 'error', error: exc.toString()});
+          } catch (err) {
+            dispatch({type: 'error', error: err.toString()});
+            this.store.dispatch(addError(err.message));
           }
         };
       }, end);
 
       return await f(...params);
     };
-
     return [call, state.submitting, state.error, state.result];
   }
 
+  /**
+   * `useBackendLazy` works fine for methods that are invoked manually (e.g. clicking a button) but may cause problems
```

Comment:
this docstring is on the wrong function..

Reply:
I think it is on the right function :P but maybe it was confusing that I started the docstring with `useBackendLazy`